pandas/core/indexes/datetimes.py

```diff
@@ -2237,6 +2238,22 @@ def date_range(start=None, end=None, periods=None, freq='D', tz=None,
     Returns
     -------
     rng : DatetimeIndex
+
+    See Also
+    ________
+    pandas.period_range : Return a fixed frequency PeriodIndex.
+    pandas.interval_range : Return a fixed frequency IntervalIndex.
+
+    Examples
+    --------
+    >>> import pandas
```

Comment:
We assume that these imports are available

```python
import pandas as pd
import numpy as np
```

so you can remove the import and do `pd.range_range`